plugins/modules/ec2_vpc_net.py

```diff
@@ -188,6 +188,11 @@
           returned: always
           type: str
           sample: pk_vpc4
+    owner_id:
+      description: The AWS account which owns the VPC
+      returned: always
+      type: str
+      sample: 123456789012
```

Comment:
```suggestion
      sample: 123456789012
      version_added: 1.4.0
```

Reply:
This has always been there, just not documented